packages/ui-kit/package.json

```diff
@@ -17,7 +17,6 @@
   "files": [
     "dist",
     "package.json",
-    "code-block-themes.js",
```

Comment:
This is in theory a breaking change. I still need to decide if we want to mark it as a breaking change or not, since this was only used by the gatsby theme.

Reply:
No, we're still in alpha releases which are always considered breaking.